Rakefile

```diff
@@ -187,7 +185,6 @@ task :check_consistency_between_aws_and_carrenza do
     govuk::apps::content_audit_tool::db::allow_auth_from_lb
     govuk::apps::content_audit_tool::db::lb_ip_range
     govuk::apps::content_audit_tool::db::rds
-    govuk::apps::content_data_admin::content_performance_manager_uri
```

Comment:
How come we don't `govuk::apps::content_data_admin::content_data_api_uri`?

Reply:
This parameter was used to override the default behaviour in Plek, so back when the content-data-admin did `Plek.find('content-performance-manager')` this parameter allowed overriding the value returned to be the content-data-api in AWS Staging and AWS Production.

Once the content-data-admin was changed to ask Plek for the content-data-api, the default behaviour is fine, so this parameter was removed [1]

1: https://github.com/alphagov/govuk-puppet/commit/9fe36303b81c748999253080945486a2af3f4d45

I guess the hieradata could have been removed then too, but it got missed.